src/components/templates/PosterHall/components/PosterPreview/PosterPreview.tsx

```diff
@@ -1,39 +1,46 @@
-import React, { useMemo } from "react";
+import React, { useCallback, useMemo } from "react";
 import classNames from "classnames";
 
 import { PosterPageVenue } from "types/venues";
 
 import { WithId } from "utils/id";
 
+import { PosterCategory } from "components/atoms/PosterCategory";
+
 import "./PosterPreview.scss";
 
 export interface PosterPreviewProps {
   posterVenue: WithId<PosterPageVenue>;
-  onClick?: () => void;
+  enterVenue: (venueId: string) => void;
 }
 
 export const PosterPreview: React.FC<PosterPreviewProps> = ({
   posterVenue,
-  onClick,
+  enterVenue,
 }) => {
   const { title, authorName, categories } = posterVenue.poster ?? {};
 
+  const venueId = posterVenue.id;
+
   const posterClassnames = classNames("PosterPreview", {
     "PosterPreview--live": posterVenue.isLive,
   });
 
+  const handleEnterVenue = useCallback(() => enterVenue(venueId), [
+    enterVenue,
+    venueId,
+  ]);
+
   const renderedCategories = useMemo(
     () =>
       Array.from(new Set(categories)).map((category) => (
-        <div key={category} className="PosterPreview__category">
-          {category}
-        </div>
+        <PosterCategory key={category} category={category} />
```

Comment:
This used `div`, whereas `PosterCategory` now uses `span`. I'm assuming that's intentional/desired? If so, all good.

Reply:
Yes